coreneuron/nrnoc/register_mech.cpp

```diff
@@ -159,6 +159,7 @@ void alloc_mech(int n) {
         int i;
         for (i = 0; i < memb_func_size_; ++i) {
             nrn_mech_data_layout_[i] = 1;
+            memb_func[i].alloc = NULL; //mark as non-instantiated
```

Comment:
`members_func` is allocated with `calloc` so it's already initialised to 0 (unless I am missing something here...)

Reply:
@pramodk this commit fixes verification of the instantiation for non-`calloc` allocation of `memb_func`

Reply:
could you check why build was failed? might need close and reopen.